schema/defs.json

```diff
@@ -114,6 +114,16 @@
                 },
                 "env": {
                     "$ref": "#/definitions/Env"
+                },
+                "timeout": {
+                    "oneOf": [
+                        {
+                            "type": "integer"
+                        },
+                        {
+                            "type": "null"
+                        }
+                    ]
```

Comment:
We may want to add this `oneOf` as `intPointer` to match our other `*Pointer` definitions.